src/codegen/riscv64/assembler-riscv64.cc

```diff
@@ -1781,6 +1798,25 @@ void Assembler::fmv_d_x(FPURegister rd, Register rs1) {
   GenInstrALUFP_rr(0b1111001, 0b000, rd, rs1, zero_reg);
 }
 
+// RV64C Standard Extension
+void Assembler::c_nop() {
+  GenInstrCI(0b000, C1, ToRegister(0), 0);
+}
+
+void Assembler::c_addi(Register rd, Register rs1, int16_t imm6) {
```

Comment:
I suggest we remove the redundant parameter `Register rs1` (and for all other arithmetic ops w/ similar interfaces).